### DISCUSSION TITLE
testing newest version

### DISCUSSION THREAD

**Discussion by eborgman** [lzachmann/models-for-missing-data#14] [General]
@lzachmann 

I'm running a model from species-richness.yml, negative binomial (this is the one that's been giving me the most trouble). Full path:
assets/uplands-output/ROMN/GRKO/species-richness/native.rich/neg-binom_exp_none_fixed-stratum/botanist-ja_mgmtzone-deflections

The model stopped and hit a browser() (the browser was the last line, not in the screen shot)

![image](https://user-images.githubusercontent.com/25830271/141206397-d6d853b5-0bf7-4e64-a4a6-155fe519ac98.png)

**Comment by lzachmann** [comment]
Came down to the cached inits. Basically, Erin changed the data to include one more management zone, which means one more Beta. And the cached inits had one too few Beta params. Suggesting something like a trycatch to attempt using defaults if the inherited / cached inits do not work. Or perhaps a dimensions test before a jags model is attempted, which would allow us to at least print a more meaningful error message.

**Reply by lzachmann** [reply]
But the stopgap solution is simply to use `initial values: default` in the config file.
